deltacat/tests/utils/test_record_batch_tables.py

```diff
@@ -0,0 +1,241 @@
+import unittest
+import pyarrow as pa
+
+from deltacat.utils.pyarrow import RecordBatchTables
+
+
+def is_sorted(batched_tables: RecordBatchTables,
```

Comment:
Minor: these methods are not reusable across different tests, so we could move them to the end of the file and start with `_` (underscore).